File: Backend/YılmazMotorOtomotivWebApi/YılmazMotorWeb.Entities/Concretes/ProductReview.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using YılmazMotorWeb.Entities.Identity;

namespace YılmazMotorWeb.Entities.Concretes
{
	public class ProductReview
	{
		public int Id { get; set; }
		public int ProductId { get; set; }
		public int UserId { get; set; }
		public string ReviewText { get; set; }
		public int Rating { get; set; } 
		public DateTime CreatedAt { get; set; } = DateTime.Now;
		public Product Product { get; set; }
		public AppUser User { get; set; }

	}
}

File: Frontend/yilmazMotorOtomotiv/src/app/Models/productReview.ts
export interface ProductReview {
  id: number;
  productId: number;
  userId: number;
  reviewText: string;
  rating: number;
  createdAt: Date;
  productName: string;
  userName: string;
}

export interface CreateProductReviewDto {
  productId: number;
  userId: number;
  reviewText: string;
  rating: number;
}
